src/botprovisioner.ts

```diff
@@ -471,6 +471,44 @@ Usage: \`listusers\``,
 Usage: \`listrooms\``,
 			withPid: false,
 		});
+		this.registerCommand("listgroups", {
+			fn: async (sender: string, param: string, sendMessage: SendMessageFn) => {
+				if (!this.bridge.hooks.listGroups) {
+					await sendMessage("Feature not implemented!");
+					return;
+				} else if (!this.bridge.groupSyncEnabled) {
+					await sendMessage("Group sync is not enabled!");
+					return;
+				}
+				const descs = await this.provisioner.getDescMxid(sender);
+				if (descs.length === 0) {
+					await sendMessage("Nothing linked yet!");
+					return;
+				}
+				let reply = "";
+				for (const d of descs) {
+					const groups = await this.bridge.hooks.listGroups(d.puppetId);
+					reply += `### ${d.puppetId}: ${d.desc}:\n\n`;
+					for (const g of groups) {
+						const mxid = await this.bridge.groupSync.getMxid({
+							puppetId: d.puppetId,
+							groupId: g.id!,
+						});
+						const replyPart = ` - ${g.name}: [${g.name}](https://matrix.to/#/${mxid})\n`;
```

Comment:
```suggestion
						const replyPart = ` - ${g.name}: [${g.name}](https://matrix.to/#/+{mxid})\n`;
```

shouldn't it be this?

Reply:
That's a javascript template string variable, not a Matrix identifier sigil

Reply:
Oh woops, you're right